spec/controllers/view_data/page_controller_spec.rb

```diff
@@ -8,5 +8,20 @@
       get :homepage
       expect(page.title).to match(/\AService Performance/)
     end
+
+    it 'has a cookie page' do
+      get :cookies
+      expect(controller).to have_rendered('view_data/pages/cookies')
+    end
+
+    it 'has a privacy policy page' do
+      get :privacy
+      expect(controller).to have_rendered('view_data/pages/privacy')
+    end
+
+    it 'has a terms and conditions page' do
+      get :terms
+      expect(controller).to have_rendered('view_data/pages/terms')
+    end
```

Comment:
Looks good. We could potentially add a feature spec to check the footer links work and contain the correct content, but not sure if that would be overkill?